terraform/environments/example/platform_locals.tf

```diff
@@ -0,0 +1,37 @@
+locals {
+
+  application_name = "example"
```

Comment:
I'm wondering if there is a way we can generate this dynamically rather than the hard coded substitution which happens during the template process

Reply:
yep, good point, I'll make a note of that for other changes to come in MP repo